src/components/RightHandMenu/DonutChart/Legend/Legend.js

```diff
@@ -65,7 +69,7 @@ function LegendItem({legendItem, index, displayValue}) {
         </div>
       </div>
       <div className="leg__rt" disabled="true" style={{color: "lightgrey"}}>
-        {displayValue}
+        {displayValue} fuck face
```

Comment:
lol it looks like some debugging text is still along for the ride.

Reply:
How embarrassing! Sorry about that. Old habits from school die hard.

Reply:
That wasn't even a comment either ... That was being printed to the front end. I think I've learned an important lesson here.